src/DayColumn.js

```diff
@@ -160,6 +161,8 @@ class DaySlot extends React.Component {
       events, entityKeyAccessor, startAccessor, endAccessor, min, totalMin: this._totalMin, step, rightOffset
     })
 
+    const eventProps = componentProps.event || {};
```

Comment:
Are `componentProps` always defined? There might be an error when trying to read `event` from `undefined`

Reply:
Yeah, they're given a default value higher in the component tree, in `Calendar.jsx`.
```
let {
       view, toolbar, events, singleDayEventsOnly
     , culture
     , components = {}
     , componentProps = {}
     , formats = {}
     , style
     , className
     , elementProps
     , date: current
     , ref
     , ...props } = this.props;
```

Reply:
oh, right. sorry I missed that.